### PR TITLE
Update README.md to point to tutorials deployment instructions

### PR DESCRIPTION
**Pull request by abbycross**
This adds a link from the README to the tutorials directory README, where instructions for tutorial files (deploying, adding, etc.) lives.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -438,6 +438,8 @@ We use [MDX](https://mdxjs.com), which is like normal markdown but adds extensio
 
 Refer to the [Common Markdown syntax](https://commonmark.org/) for a primer on Markdown. The below guide focuses on the other features you can use when writing docs.
 
+If you need to deploy a tutorial on the Learning Platform, see the instructions in the [tutorials directory README](https://github.com/Qiskit/documentation/blob/main/tutorials/README.md).
```
Comment (abbycross) [inline README.md:441]:
```suggestion
If you need to deploy an open-access tutorial on the Learning Platform, see the instructions in the [tutorials directory README](https://github.com/Qiskit/documentation/blob/main/tutorials/README.md).
```

File: README.md [hunk 2]
```diff
@@ -438,6 +438,8 @@ We use [MDX](https://mdxjs.com), which is like normal markdown but adds extensio
 
 Refer to the [Common Markdown syntax](https://commonmark.org/) for a primer on Markdown. The below guide focuses on the other features you can use when writing docs.
 
+If you need to deploy a file from the `/tutorials/` directory to the Learning Platform, see the instructions in the [tutorials directory README](https://github.com/Qiskit/documentation/blob/main/tutorials/README.md).
```
Comment (Eric-Arellano) [inline README.md:441]: What about instead having a new section `## How to deploy docs` before `# How to write the documentation`? For normal docs, it links to https://github.ibm.com/IBM-Q-Software/iqp-channel-docs#syncing-content-with-open-source-repo. For tutorials, it has what you wrote

Comment (abbycross) [reply README.md:441]: Sounds good!

File: README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
 # Qiskit docs
 
-The documentation content home for https://docs.quantum.ibm.com (excluding API reference).
+The documentation content home for [https://docs.quantum.ibm.com](https://docs.quantum.ibm.com) (excluding API reference).
```
Comment (Eric-Arellano) [inline README.md:3]: Shouldn't be necessary. It already gets detected as a link.

File: README.md [hunk 4]
```diff
@@ -76,7 +76,7 @@ npm install
 You can preview the docs locally by following these two steps:
 
 1. Ensure Docker is running. For example, open Rancher Desktop.
-2. Run `./start` in your terminal, and open http://localhost:3000 in your browser.
+2. Run `./start` in your terminal, and open `http://localhost:3000` in your browser.
```
Comment (Eric-Arellano) [inline README.md:79]: Why? It's useful that you could click the link

File: README.md [hunk 5]
```diff
@@ -97,7 +97,7 @@ To use live previews, push your branch to `upstream` rather than your fork. GitH
 
 ## Staging
 
-We also re-deploy the docs every time we merge into `main` at the site https://qiskit-docs-preview-staging.1799mxdls7qz.us-south.codeengine.appdomain.cloud.
+We also re-deploy the docs every time we merge into `main` at the site `https://qiskit-docs-preview-staging.1799mxdls7qz.us-south.codeengine.appdomain.cloud`.
```
Comment (Eric-Arellano) [inline README.md:100]: Ditto, why? It's useful to click the link

File: README.md [hunk 6]
```diff
@@ -380,7 +385,7 @@ In this case, no commit will be automatically created.
 This is useful when new docs content is published, usually corresponding to new releases or hotfixes for content issues. If you're generating a patch release, also see the below subsection for additional steps.
 
 1. Choose which documentation you want to generate (`qiskit`, `qiskit-ibm-provider`, `qiskit-ibm-runtime`, or `qiskit-transpiler-service`) and its version.
-2. Determine the full version, such as by looking at https://github.com/Qiskit/qiskit/releases
+2. Determine the full version, such as by looking at <https://github.com/Qiskit/qiskit/releases>
```
Comment (Eric-Arellano) [inline README.md:388]: Why? ditto the other links

File: README.md [hunk 7]
```diff
@@ -430,6 +435,18 @@ The add the following to your `.gitconfig` (usually found at `~/.gitconfig`).
   textconv = sh -c 'sphobjinv convert plain "$0" -'
 ```
 
+## How to deploy docs
+
+### Docs (content that lives on IBM Quantum Platform)
```
Comment (Eric-Arellano) [inline README.md:440]:
```suggestion
### Docs (content that lives on docs.quantum.ibm.com)
```

File: README.md [hunk 8]
```diff
@@ -430,6 +435,18 @@ The add the following to your `.gitconfig` (usually found at `~/.gitconfig`).
...
+
+See [https://github.ibm.com/IBM-Q-Software/iqp-channel-docs#syncing-content-with-open-source-repo](https://github.ibm.com/IBM-Q-Software/iqp-channel-docs#syncing-content-with-open-source-repo).
+
+### Tutorials (content that lives on the Learning Platform
```
Comment (Eric-Arellano) [inline README.md:444]:
```suggestion
### Tutorials (content that lives on the learning.quantum.ibm.com)
```

**Review by Eric-Arellano — COMMENTED** [review]
Thanks!

**Comment by abbycross** [comment]
FYI - All the URL changes I had made are because CI failed for detecting "bare" URLs.

**Review by frankharkins — APPROVED** [review]

**Comment by frankharkins** [comment]
Weird about prettier; maybe the version in CI has changed?

**Comment by Eric-Arellano** [comment]
Idk what was going on with the formatting. Replaced by https://github.com/Qiskit/documentation/pull/1450.

**Pull request closed without merging**